### ISSUE TITLE
Slow on MacOS

### ISSUE THREAD

**Issue by ghost** [gustavklopp/LingL#42]
I installed the app using Docker but using it is difficult because it is really slow. I am waiting around 30 seconds after clicking a button to go to the next page. If I want to add a new text, I get stuck: 
<img width="258" alt="image" src="https://user-images.githubusercontent.com/46455311/183264089-852d57fa-78d9-4d2d-a04d-25a25b226d75.png">
Nothing happens and I can't proceed. I'm using an M1 Macbook Air running Big Sur.

**Comment by hieutran-coder** [comment]
Yes, me too, very slow, does anyone know how to improve?. Thank you
